### ISSUE TITLE
Lander's "Set Rotation" can't change "Up" for spaceplanes.

### ISSUE THREAD

**Issue by Ezriilc** [Ezriilc/HyperEdit#11]
If using the Lander with a spaceplane, or horizontally oriented ship, and setting "Set Rotation", it will point the nose of the craft to the sky, as if it were a vertical rocket.

**Comment by Ezriilc** [comment]
There should be check/text boxes for options such as which direction to point / which way is up / etc..

**Comment by fronbow** [comment]
I think this is a KSP thing. In the source we point the craft "up" as far as KSP thinks "up" is for the current vessel. I'll do some testing to see if there's another way of orienting a craft.
It may be that we need to check for rockets vs planes and do different ways of up.

**Comment by fronbow** [comment]
I have some ideas on how we might solve this, so today I will mostly be watching KSP in slow motion (breakpoints) and wrapping up xmas pressies!

**Comment by Ezriilc** [comment]
It has been suggested that we consider a setting to point the craft a certain way on landing.  I like the idea, so I've added it here.

**Comment by fronbow** [comment]
I would've made a new issue so we could tag it as a feature request!
Anyhow, I'm curious about what you mean?
Up or Down would be simple enough or do you/they mean setting a direction (0 - 360deg)?

**Comment by Ezriilc** [comment]
This issue is already somewhat a suggestion.  I assumed this addition would be part of any "fix" for this existing issue.  If it turns out that entirely different code/tweaks are needed, then yea, it makes sense to move this to a new issue.

Re: Up/Down/0-359...:  Yes!
It would be ideal to have all 3 attitude axes definable, and have some presets like "Current", "Up", "Down", N,S,E,W, etc..  I pictured "Current" being default.

**Comment by fronbow** [comment]
I'll put this on my todo list ;)
It's starting to feel like we need to rethink some parts of the UI (the lander panel could get quite long with this!!)

**Comment by Ezriilc** [comment]
I was just thinking the same thing.  I assume we can go wider, and/or have tabs/scrolling/whatever...?

**Comment by fronbow** [comment]
Yes, we can go wider, maybe 2+ columns. I know how scrolling works now! Just trying to find stuff on tabs or other ways to lay out controls.

**Comment by Ezriilc** [comment]
This issue is related to https://github.com/Ezriilc/HyperEdit/issues/35
